skel/.mozilla/firefox/jzlvae43.default/prefs.js

```diff
@@ -1,104 +0,0 @@
-user_pref("extensions.pendingOperations", false);
-user_pref("extensions.update.enabled", false);
-user_pref("extensions.update.notifyUser", false);
-user_pref('extensions.zotero.firstRun2', false);
-user_pref('extensions.zotero.firstRunGuidanceShown.toolbarButton', true);
-user_pref('extensions.zotero.integration.useClassicAddCitationDialog', true);
-user_pref('extensions.zoteroOpenOfficeIntegration.installed', true);
-user_pref("extensions.zoteroOpenOfficeIntegration.skipInstallation", true);
-user_pref("flagfox.warn.tld", false);
-user_pref("flagfox.warn.updates", false);
-user_pref("gfx.blacklist.direct2d", 3);
-user_pref("gfx.blacklist.layers.direct3d10", 3);
-user_pref("gfx.blacklist.layers.direct3d10-1", 3);
-user_pref("gfx.blacklist.layers.direct3d9", 3);
-user_pref("gfx.blacklist.layers.opengl", 3);
-user_pref("gfx.blacklist.webgl.angle", 3);
-user_pref("gfx.blacklist.webgl.opengl", 3);
-user_pref("intl.charsetmenu.browser.cache", "ISO-8859-1, UTF-8");
-user_pref("network.cookie.prefsMigrated", true);
-user_pref("network.websocket.enabled", false);
-user_pref("places.history.expiration.transient_current_max_pages", 97062);
-user_pref("plugin.disable_full_page_plugin_for_types", "application/msword");
-user_pref("plugin.soname.list", "libXt.so.6:libXext.so.6");
-user_pref("pref.downloads.disable_button.edit_actions", false);
-user_pref("privacy.clearOnShutdown.cookies", false);
-user_pref("privacy.clearOnShutdown.downloads", false);
-user_pref("privacy.clearOnShutdown.formdata", false);
-user_pref("privacy.clearOnShutdown.history", false);
-user_pref("privacy.clearOnShutdown.passwords", true);
-user_pref("privacy.cpd.offlineApps", true);
-user_pref("privacy.cpd.siteSettings", true);
-user_pref("privacy.item.offlineApps", true);
-user_pref("privacy.sanitize.didShutdownSanitize", true);
-user_pref("privacy.sanitize.migrateFx3Prefs", true);
-user_pref("privacy.sanitize.sanitizeOnShutdown", true);
-user_pref("privacy.sanitize.timeSpan", 0);
-user_pref("privacy.donottrackheader.enabled", true);
-user_pref("security.OCSP.URL", "");
-user_pref("security.OCSP.signingCA", "Builtin Object Token:IPS CLASE1 root");
-user_pref("security.disable_button.openCertManager", false);
-user_pref("security.warn_viewing_mixed", false);
-user_pref("signon.rememberSignons", false);
```

Comment:
Might be a good idea to have this as a lockPref - I think this is needed to stop passwords being saved